site/_docs/history.md

```diff
@@ -29,17 +29,93 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 <!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.36.0">1.36.0</a> / XXXX-XX-XX
-{: #v1-36-0}
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.37.0">1.37.0</a> / XXXX-XX-XX
+{: #v1-37-0}
 
-This release comes x months after [1.35.0](#v1-35-0),
+This release comes x months after [1.36.0](#v1-36-0),
 contains contributions from x contributors, and resolves x issues.
 
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-37-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 21.0 to 32.1.3-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-37-0}
+
+#### Dependency version upgrade
+{: #dependency-1-37-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-37-0}
+
+#### Build and test suite
+{: #build-1-37-0}
+
+#### Web site and documentation
+{: #site-1-37-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.36.0">1.36.0</a> / 2023-11-XX
+{: #v1-36-0}
+
+This release comes 3 months after [1.35.0](#v1-35-0),
+contains contributions from 30 contributors, and resolves 125 issues.
+
+Among other new features, it's worth highlighting the adding of 30 new SQL functions in various libraries such as BigQuery and Spark, many improvements hardening `TABLESAMPLE` , and also the following features:
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5615">CALCITE-5615</a>]
+ Run SQL Logic Test suite using Calcite's HSQLDB JDBC adapter
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-129">CALCITE-129</a>]
+ Support recursive `WITH` queries
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6022">CALCITE-6022</a>]
+ Support `CREATE TABLE ... LIKE DDL` in server module
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5962">CALCITE-5962</a>]
+ Support parse Spark-style syntax `LEFT ANTI JOIN` in Babel parser
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5184">CALCITE-5184</a>]
+ Support `LIMIT start, ALL` in MySQL conformance, equivalent to `OFFSET start`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5889">CALCITE-5889</a>]
+ Add the RelRule that converts `Minus` into `UNION ALL..GROUP BY...WHERE`
+
+Contributors to this release:
+Benchao Li (release manager),
+Cancai Cai,
+Claude Brisson,
+Evgeniy Stanilovskiy,
+Hanumath Maduri,
+Hongyu Guo,
+Itiel Sadeh,
+Jerin John,
+Jiajun Xie,
+Julian Hyde,
+Kaustubh Beedkar,
+Lei Shen,
+Leonid Chistov,
+Michael Mior,
+Mihai Budiu,
+Mingcan Wang,
+Oliver Lee,
+Ran Tao,
+Ruben Quesada Lopez,
+Runkang He,
+Tanner Clary,
+Thomas Rebele,
+Tim Nieradzik,
+Wegdan Ghazi,
+Wenrui Meng,
+Xiaogang Zhou,
+ZhangJian He,
+Zhengqiang Duan,
+Zoltan Haindrich,
+zoovwang.
+
```

Comment:
I'm tempted to write (and commit) a script to generate the 'contributors to this release' section. It could be a sed script that converts email addresses into people's real names.

Reply:
When I handled the contributor name, there were four cases:
* the name in commit message
* the name on Github profile
* the name in Jira account
* fails all above ways, I'll ask the contributor for it

Do you mean we can add a script to generate names just from the first case?

Reply:
Turns out that git's `.mailmap` file can do a lot of this. I've logged CALCITE-6125 to add a `.mailmap`.